res/values/strings.xml

```diff
@@ -21,6 +21,13 @@
 	<string name="ranked_status_wip"><font color="#FFAC64">Work in Progress</font></string>
 	<string name="ranked_status_graveyard"><font color="#FFFFFF">Graveyard</font></string>
 
+	<!--Block area manager fragment-->
+	<string name="block_area_editor_fragment_add">Add area</string>
+	<string name="block_area_editor_fragment_reset">Reset</string>
+	<string name="block_area_editor_fragment_done">Done</string>
+	<string name="block_area_preference_title">Customize input block areas</string>
+	<string name="block_area_preference_summary">Add, remove, and resize input block areas</string>
```

Comment:
```suggestion
<string name="block_area_preference_title">Manage input block areas</string>
	<string name="block_area_preference_summary">Manage specific areas within the screen where presses will be ignored during gameplay</string>
```
Maybe? The current description does not explain what an "input block area" does.